deployment/ccip/changeset/cs_rmn_curse_uncurse.go

```diff
@@ -0,0 +1,275 @@
+package changeset
+
+import (
+	"encoding/binary"
+	"fmt"
+	"slices"
+
+	"github.com/pkg/errors"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	commoncs "github.com/smartcontractkit/chainlink/deployment/common/changeset"
+)
+
+func GlobalCurseSubject() Subject {
+	return Subject{0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01}
+}
+
+type RMNCurseAction struct {
+	ChainSelector  uint64
+	SubjectToCurse Subject
+}
+
+type CurseAction func(e deployment.Environment) []RMNCurseAction
+
+type RMNCurseConfig struct {
+	MCMS         *MCMSConfig
+	CurseActions []CurseAction
+	Reason       string
+}
+
+func (c RMNCurseConfig) Validate(e deployment.Environment) error {
+	state, err := LoadOnchainState(e)
+
+	if err != nil {
+		return errors.Errorf("failed to load onchain state: %v", err)
```

Comment:
nit: we're trying to actively discontinue the use of `errors` in favor of `fmt` - so here `fmt.Errorf` instead

Reply:
@makramkd I had to change this to make the linter pass, everything was using fmt.Errorf but linting would fail

Reply:
I can't find the previous run, it will probably be quicker to change the format once more and see if the linting issues reappear, anyways from what I get we should probably update the linting rule if this flags fmt.Errorf